tests/test_remote_inventory.py

```diff
@@ -45,9 +58,11 @@ def test_compile_fetch(self):
             temp_output,
             "--inventory-path",
             temp_inv,
-            "-t",
+            "--targets",
             "remoteinv-example",
             "remoteinv-nginx",
+            "-p",
```

Comment:
we usually don't add parallelism options in tests. Is this to test fetching remote invs with parrallel threads?
If yes then it might be better added in an explicit test for that. 
If you want to make your local tests faster, you could add this parallelism option in your local `.kapitan` config to apply on all kapitan projects? :)